assets/k8sclusterinfo.yaml

```
@@ -171,3 +174,25 @@ k8scluster:
         size:
           min: 10
           max: 40
+  ncpvpc:
```

```suggestion
  ncp:
```

Minor fix. Since we do not use `ncpvpc` term anymore. ([see the note about nameing](https://github.com/cloud-barista/cb-tumblebug/blob/main/docs/migration_guide_0.11.1_to_0.11.8.md#-csp-provider-%EB%AA%85%EC%B9%AD-%EB%B3%80%EA%B2%BD-breaking-change))

but you need to test again.
- fetch&rebase your cb-spider dev environment. (check all terms for `ncpvpc` in the repo)
- fetch&rebase your cb-tumblebug dev environment. (check all terms for `ncpvpc` in the repo)
- run cb-tumblebug with the [new version](https://github.com/cloud-barista/cb-tumblebug/blob/main/docs/migration_guide_0.11.1_to_0.11.8.md#-%EC%82%AC%EC%A0%84-%EC%A4%80%EB%B9%84-%ED%95%84%EC%88%98)
- run test again. if it pass your test, let me know through this PR.